ginga/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = 'ginga' %}
 {% set version = '2.5.20160627100500' %}
```

Comment:
He didn't update the version number. Does that mean we cannot update this until he does?